node/router/src/heartbeat.rs

```diff
@@ -290,11 +299,11 @@ pub trait Heartbeat<N: Network>: Outbound<N> {
     /// This function attempts to connect to any disconnected trusted peers.
     fn handle_trusted_peers(&self) {
         // Ensure that the trusted nodes are connected.
-        for peer_ip in self.router().trusted_peers() {
+        for peer_addr in self.router().trusted_peers() {
             // If the peer is not connected, attempt to connect to it.
-            if !self.router().is_connected(peer_ip) {
-                // Attempt to connect to the trusted peer.
-                self.router().connect(*peer_ip);
+            if !self.router().is_connected(peer_addr) {
+                debug!("Attempting to (re-)connect to trusted peer `{peer_addr}`");
```

Comment:
I added this log message, so it is a little easier to understand what is going on when running the devnet. Otherwise you see a lot of "connection refused" messages, but it is unclear why.